src/main/resources/mycila/APACHE-2.txt

```diff
@@ -1,4 +1,4 @@
-Copyright (C) ${project.inceptionYear}-${currentYear} ${owner}
+Copyright (C) ${license.git.copyrightYears} ${owner}
```

Comment:
Is this a backwards compatible change? I'm guessing it is but just checking - if I moved my downstream project to use this new version but didn't change anything it would "just work" right?